specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json

```diff
@@ -2760,6 +2792,28 @@
       },
       "description": "The gallery artifact version source."
     },
+    "ExecutedValidation": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "This property specifies the type of image version validation."
+        },
+        "status": {
+          "type": "string",
+          "description": "This property specifies the status of the validationProfile of the image version."
+        },
+        "version": {
+          "type": "string",
+          "description": "This property specifies the valid version of the validation."
+        },
+        "executionTime": {
+          "type": "string",
+          "format": "date-time",
```

Comment:
Are users really required to input the string as the date-time format as seen here under String Formats?
https://swagger.io/docs/specification/data-models/data-types/ 

How does the user easily get this value? Seems like something they would need to copy and paste.

Reply:
this is a DateTimeOffset data type